src/publishers/github.js

```diff
@@ -31,7 +31,8 @@ export default async (artifacts, packageJSON, forgeConfig, authToken, tag) => {
           repo: forgeConfig.github_repository.name,
           tag_name: tag || `v${packageJSON.version}`,
           name: tag || `v${packageJSON.version}`,
-          draft: true,
+          draft: forgeConfig.github_repository.draft !== false,
```

Comment:
> I'd be willing to help improve the publishers with the right direction if you're interested.

People interested in contributing are very welcome 😄 
@MarshallOfSound is going to work on the CLI feature he proposed in a separate PR.